src/main/java/mate/academy/Main.java

```diff
@@ -2,16 +2,29 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Injector;
+import mate.academy.lib.Service;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.security.AuthenticationService;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
+import mate.academy.service.ShoppingCartService;
 
 public class Main {
+    private static final Injector instance = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+
+        @Inject
+        AuthenticationService authenticationService;
+        @Service
         MovieService movieService = null;
+        @Service
+        ShoppingCartService shoppingCartService = null;
```

Comment:
create `User`, register it, login it, call methods from `shoppingCartService`

```suggestion
        AuthenticationService authenticationService = (
                AuthenticationService) instance.getInstance(AuthenticationService.class);
        ShoppingCartService shoppingCartService = (
                ShoppingCartService) instance.getInstance(ShoppingCartService.class);
```